store.go

```diff
@@ -1515,6 +1516,12 @@ func (s *store) ImageBigData(id, key string) ([]byte, error) {
 		if err == nil {
 			return data, nil
 		}
+		if store.Exists(id) {
+			foundImage = true
+		}
+	}
+	if foundImage {
```

Comment:
should this be `if !foundImage {`  ?

Reply:
That wouldn't match the error that we return in the block.

Reply:
ugh, coulda sworn that said 'error locating image...'  Darn dyslexia!  Thx, all good.